nilearn/decomposition/dict_learning.py

```diff
@@ -188,13 +200,11 @@ def _init_dict(self, data):
         self.components_init_ = components
 
     def _init_loadings(self, data):
-        self._loadings_init = self._cache(_compute_loadings,
-                                          func_memory_level=2)(
-            self.components_init_,
-            data)
+        self._loadings_init = self._cache(_compute_loadings)(
+            self.components_init_, data)
 
     def fit(self, imgs, y=None, confounds=None):
-        """Compute the mask and the ICA maps across subjects
+        """Compute the mask and component maps across subjects
```

Comment:
this is one of the many issues with the existing code. not introduced by
the proposed patch ...

On Tue, Mar 29, 2016 at 6:01 PM, Alexandre Abadie <notifications@github.com>
wrote:

> In nilearn/decomposition/dict_learning.py
> <https://github.com/nilearn/nilearn/pull/1067#discussion_r57749736>:
>
> >
> >      def fit(self, imgs, y=None, confounds=None):
> > -        """Compute the mask and the ICA maps across subjects
> > +        """Compute the mask and component maps across subjects
>
> Period missing ?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/nilearn/nilearn/pull/1067/files/532c2da19d52928e94a99d6ac1f8dfa1d65067d2#r57749736>
>



-- 
Elvis Dohmatob,
*sent from my  potato...*
http://dohmatob.blogspot.fr/, https://team.inria.fr/parietal/elvis/,
https://github.com/dohmatob